### ISSUE TITLE
ValueError: Shapes (1, 1, 1024, 255) and (21, 1024, 1, 1) are incompatible

### ISSUE THREAD

**Issue by aless2003** [OlafenwaMoses/ImageAI#589]
So basically I trained a YOLOv3 Model with ImageAI.
During the load() function it crashes and gives back this traceback:

`Traceback (most recent call last):
  File "C:/Users/aless/PycharmProjects/Mikulyzer_Locate/firstDetection.py", line 10, in <module>
    detector.loadModel()
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\imageai\Detection\__init__.py", line 197, in loadModel
    model.load_weights(self.modelPath)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\keras\engine\saving.py", line 492, in load_wrapper
    return load_function(*args, **kwargs)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\keras\engine\network.py", line 1230, in load_weights
    f, self.layers, reshape=reshape)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\keras\engine\saving.py", line 1237, in load_weights_from_hdf5_group
    K.batch_set_value(weight_value_tuples)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\keras\backend\tensorflow_backend.py", line 2960, in batch_set_value
    tf_keras_backend.batch_set_value(tuples)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\tensorflow\python\keras\backend.py", line 3066, in batch_set_value
    assign_op = x.assign(assign_placeholder)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py", line 1141, in assign
    self._shape.assert_is_compatible_with(value_tensor.shape)
  File "C:\Users\aless\anaconda3\envs\Mikulyzer_Locate\lib\site-packages\tensorflow\python\framework\tensor_shape.py", line 1103, in assert_is_compatible_with
    raise ValueError("Shapes %s and %s are incompatible" % (self, other))`

My Code is:

> execution_path = os.getcwd()
> detector = ObjectDetection()
> detector.setModelTypeAsYOLOv3()
> detector.setModelPath(os.path.join(execution_path, "detection_model-ex-013--loss-0012.906.h5"))
> detector.loadModel() # crashes here

Basically, did I not train it right? Or do I not load it correctly?

**Comment by OlafenwaMoses** [comment]
Hi @aless2003 , You forgot to specify the path to the the `detection_config.json` generated during the training.

```
detector.setJsonPath("detection_config.json")
detector.loadModel()
```

**Comment by OlafenwaMoses** [comment]
Follow this link to learn more

https://imageai.readthedocs.io/en/latest/customdetection/index.html

**Comment by aless2003** [comment]
Thanks for the fast reply. 
Unfortunately Python says ObjectDetection doesn't have something called setJsonPath. 
I seem to be on the most recent version though.

**Comment by OlafenwaMoses** [comment]
Remove `from imageai.Detection import ObjectDetection` and use the import below instead.

```
from imageai.Detection.Custom import CustomObjectDetection
```

**Comment by aless2003** [comment]
Oh! Alright now I understand. Thank you very much! I'm sorry I took your time for that, it works now. 

Have a beautiful day!
